src/emulate.c

```diff
@@ -615,7 +504,26 @@ void rv_step(riscv_t *rv, int32_t cycles)
         }
         last_pc = rv->PC;
 
-        /* execute the block */
+#if RV32_HAS(JIT)
+        /* execute the block by JIT compiler */
+        exec_block_func_t code = NULL;
```

Comment:
Move tyepdef `exec_block_func_t` here.

Reply:
I typedef it to the top of function `rv_step`, because formatter format it to `typedef bool (*exec_block_func_t)(riscv_t * rv, rv_insn_t * ir);` if we tyepdef here.